tests/tests_semantics.py

```diff
@@ -1,19 +1,17 @@
 from compiler.semantics import check_yaml_syntax
 import unittest
+from yamllint.config import YamlLintConfig
+from yamllint import linter
 
 class MyTest(unittest.TestCase):
-    def test(self):
-        input = """
-        lightweight_components:
-  - type: compute_element
-    repository_url: "https://github.com/WLCG-Lightweight-Sites/wlcg_lightweight_site_ce_cream"
-    nodes:
-      - node: lw-site-droplet-0
-        container_count: 1
-      - node: ec2-18-184-37-92.eu-central-1
-        container_count: 2
-    preferred_tech_stack:
-      level_2_configuration: yaim
-      """    
+  conf = YamlLintConfig('extends: relaxed')
 
-        self.assertEqual(input, input)
\ No newline at end of file
+  def test_check_yaml_syntax(self):   
+    augmented_yaml_file = check_yaml_syntax('../.temp/runtime.yaml')
```

Comment:
@pumpkinman008 The ./temp/runtime.yaml does not exist before running the compiler. Can you copy it in the tests/data package instead and use the one in there for testing? That way we are sure that CI would be happy when we move to one.